### ISSUE TITLE
Interactive Swagger does not apply credentials on 303 See Other from successful POST /ctms

### ISSUE THREAD

**Issue by jwhitlock** [mozilla-it/ctms-api#105]
When submitting a ``POST /ctms`` with this content:

```json
{
  "email": {
    "primary_email": "test-20210315@example.com",
    "email_id": "f6575c19-ae5b-428c-a2cc-74bb6ffd40c2"
  }
}
```

The Swagger UI returns a ``401 Unauthorized``:

```json
{
  "detail": "Could not validate credentials"
}
```

However, fetching the newly created record (``GET /ctms/f6575c19-ae5b-428c-a2cc-74bb6ffd40c2``) succeeds.

If it is done locally, you can see the ``303 See Other`` redirect from the server. It appears that the Swagger UI is following the redirect, but not adding the credentials to the new fetch, so it is impossible to tell if there was a credential issue or it was successful.

**Comment by imbstack** [comment]
I guess it sorta makes sense to strip auth header before following a redirect.

**Comment by jwhitlock** [comment]
Would you be OK with a 20x response, probably `201 Created`, with a `Location` header and the contact JSON as content?

**Comment by bsieber-mozilla** [comment]
I think this can be closed, since the merge of pr #189
